cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -220,6 +220,15 @@ func ReadStaticPodFromDisk(manifestPath string) (*v1.Pod, error) {
 
 // LivenessProbe creates a Probe object with a HTTPGet handler
 func LivenessProbe(host, path string, port int, scheme v1.URIScheme) *v1.Probe {
+	return createHTTPProbe(host, path, port, scheme, 15, 15, 8, 10)
+}
+
+// ReadinessProbe creates a Probe object with a HTTPGet handler
+func ReadinessProbe(host, path string, port int, scheme v1.URIScheme) *v1.Probe {
+	return createHTTPProbe(host, path, port, scheme, 0, 15, 3, 1)
+}
+
+func createHTTPProbe(host, path string, port int, scheme v1.URIScheme, initialDelaySeconds int32, timeoutSeconds int32, failureThreshold int32, periodSeconds int32) *v1.Probe {
```

Comment:
should be:
`initialDelaySeconds, timeoutSeconds, failureThreshold, periodSeconds int32`